src/DataTable/Table.tsx

```diff
@@ -55,9 +56,22 @@ const StyledTable = styled.table<React.ComponentPropsWithoutRef<'table'>>`
 
   .TableHeader,
   .TableCell {
+    text-align: start;
     border-bottom: 1px solid ${get('colors.border.default')};
   }
 
+  .TableHeader[data-cell-align='end'],
+  .TableCell[data-cell-align='end'] {
+    text-align: end;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .TableHeader[data-cell-align='end'] > button {
```

Comment:
Could we add a class to the table sort button instead of using an element selector?

Reply:
Sure!